examples/Makefile

```diff
@@ -0,0 +1,34 @@
+L=/home/arnetheduck/src/nlvm/ext/llvm-8.0.0.src/sha/bin
```

Comment:
Could you fix these paths?

Reply:
I could if I had a good idea how that didn't involve lots of mess :) WIP for now, this branch shouldn't be merged yet